app/code/Magento/Catalog/Block/Product/View/Options.php

```diff
@@ -20,6 +20,11 @@
  */
 class Options extends \Magento\Framework\View\Element\Template
 {
+    /**
+     * Option type percent
+     */
+    protected static $typePercent = 'percent';
```

Comment:
Pleare reuse \Magento\Catalog\Model\Product\Option\Value::TYPE_PERCENT instead of introducing this property